src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,155 @@
+    }
+
+    @Test
+    void age_null_notOk() {
+        actualuser.setAge(null);
+        assertThrows(RuntimeException.class, () -> registrationService.register(actualuser));
+    }
+
+    @Test
+    void age_0_notOk() {
+        actualuser.setAge(0);
+        assertThrows(RuntimeException.class, () -> registrationService.register(actualuser));
+    }
+
+    @Test
+    void age_less0_notOk() {
+        actualuser.setAge(-25);
+        assertThrows(RuntimeException.class, () -> registrationService.register(actualuser));
+    }
+
+    @Test
+    void age_maxint_Ok() {
+        actualuser.setAge(2147483647);
+        User expectedUser = registrationService.register(actualuser);
+        assertEquals(expectedUser, actualuser);
+        assertEquals(storageDao.get(expectedUser.getLogin()), actualuser);
+        assertEquals(Storage.people.size(), 1);
+    }
+
+    @Test
+    void loginLength_6_Ok() {
+        actualuser.setLogin("123456");
+        User expectedUser = registrationService.register(actualuser);
+        assertEquals(expectedUser, actualuser);
+        assertEquals(storageDao.get(expectedUser.getLogin()), actualuser);
+        assertEquals(Storage.people.size(), 1);
+    }
+
+    @Test
+    void loginLength_more6_Ok() {
+        actualuser.setLogin("12345678009");
+        User expectedUser = registrationService.register(actualuser);
+        assertEquals(expectedUser, actualuser);
+        assertEquals(storageDao.get(expectedUser.getLogin()), actualuser);
+        assertEquals(Storage.people.size(), 1);
+    }
+
+    @Test
+    void loginLength_less6_notOk() {
+        actualuser.setLogin("12345");
+        assertThrows(RuntimeException.class, () -> registrationService.register(actualuser));
+    }
+
+    @Test
+    void loginLength_0_notOk() {
+        actualuser.setLogin("");
+        assertThrows(RuntimeException.class, () -> registrationService.register(actualuser));
+    }
+
+    @Test
+    void loginLength_null_notOk() {
+        actualuser.setLogin(null);
+        assertThrows(RuntimeException.class, () -> registrationService.register(actualuser));
+    }
+
+    @Test
+    void passwordLength_more6_Ok() {
+        actualuser.setPassword("12345678009");
+        User expectedUser = registrationService.register(actualuser);
+        assertEquals(expectedUser, actualuser);
+        assertEquals(storageDao.get(expectedUser.getLogin()), actualuser);
+        assertEquals(Storage.people.size(), 1);
+    }
+
+    @Test
+    void passwordLength_less6_notOk() {
+        actualuser.setPassword("12345");
+        assertThrows(RuntimeException.class, () -> registrationService.register(actualuser));
+    }
+
+    @Test
+    void passwordLength_6_notOk() {
+        actualuser.setPassword("123456");
+        User expectedUser = registrationService.register(actualuser);
+        assertEquals(expectedUser, actualuser);
+        assertEquals(storageDao.get(expectedUser.getLogin()), actualuser);
+        assertEquals(Storage.people.size(), 1);
```

Comment:
The test method name `passwordLength_6_notOk` does not follow the naming convention and the expected behavior is incorrect. Since the minimum password length is 6, this should be an OK scenario. Rename the method to `register_passwordLength6_Ok` and ensure that the test is correctly expecting a successful registration.